sql/src/main/java/io/crate/planner/IsStatementExecutionAllowed.java

```diff
@@ -116,8 +109,7 @@ public Boolean visitQueriedSelectRelation(QueriedSelectRelation relation, Void c
 
         @Override
         public Boolean visitQueriedTable(QueriedTable<?> queriedTable, Void context) {
-            TableInfo tableInfo = queriedTable.tableRelation().tableInfo();
-            return isSysSchema(tableInfo.ident().schema());
+            return process(queriedTable.tableRelation(), context);
```

Comment:
how will this behave if the schema is different than `doc` but still different than `sys` or `information_schema`?
I guess it will allow it, but is this the correct behavior?

Reply:
It will allow any system table (sys, information_schema, pg_catalog), but no user tables (doc or custom schema)